src/components/NewOrganizationModal/NewOrganizationModal.jsx

```diff
@@ -50,7 +49,8 @@ const NewOrganizationModal = ({ isOpen, onDismiss, onSubmit }) => {
             aria-describedby="modal-input-for-org-descp"
             id="modal-input-for-org"
             value={formik.values.newOrganizationSuggestion}
-            autoFocus
+            // eslint-disable-next-line jsx-a11y/no-autofocus
+            autoFocus // IMPORTANT we should reconsider autofocus use. See: https://trello.com/c/4pe1zgS9/1331-accessibility-linting-issues-deferred
```

Comment:
Hurray jsx-a11y lint plugin. It wasnt catching anything previously. We have it back!

I have no idea how important removing autofocus is. I consider exploring that firmly out of scope of this refactor.